.github/workflows/rav1e.yml

```diff
@@ -280,9 +278,14 @@ jobs:
         $NASM_VERSION="nasm-2.14.02"
         curl -LO "https://people.xiph.org/~tdaede/$NASM_VERSION-win64.zip"
         7z e -y "$NASM_VERSION-win64.zip" -o"C:\nasm"
-    - name: Set environment variables
+        echo "::add-path::C:\nasm"
+    - name: Set MSVC x86_64 linker path
+      shell: pwsh
       run: |
-        echo "::add-path::$Env:VS_PATH\$Env:COMPILER_SUBPATH;C:\nasm"
+        $LinkGlob = "VC\Tools\MSVC\**\bin\Hostx64\x64\link.exe"
+        Set-Alias vswhere "${env:ProgramFiles(x86)}\Microsoft Visual Studio\Installer\vswhere.exe"
```

Comment:
Why not just add this to PATH temporarily and then call vswhere? seems a bit unnecessary to set an alias

Reply:
The path suggestion is certainly more terse. Whether it is clearer than setting an alias is less certain.
I would rather not add another network dependency for a binary that is installed in the base image, just to find another binary also in the base image.

Reply:
Main reason why I suggested it was because usually, if github actions is up, github is probably also up, and since my personal preference is to not to rely on pre installed stuff on a machine unless the location is explicitly stated. Plus, that way the build would easily be reproducible on my local machine just by running the same exact commands